saleor/graphql/webhook/tests/mutations/test_webhook_dry_run.py

```diff
@@ -605,7 +605,6 @@ def async_subscription_webhooks_with_root_objects(
     }
 
 
-# TODO will be fixed in PR refactoring the mutation
 @pytest.mark.skip
```

Comment:
Removed comment, but test is still skipped. Is it intended?

Reply:
Nope. Thanks!